pkg/controller/storagecapabilities.go

```diff
@@ -53,13 +56,49 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
 	"csi.trident.netapp.io/ontap-san": {{AccessMode: v1.ReadWriteOnce, VolumeMode: v1.PersistentVolumeBlock}},
 }
 
-// Get finds and returns a predefined StorageCapabilities for a given StorageClass
-func Get(sc *storagev1.StorageClass) ([]StorageCapabilities, bool) {
+// GetStorageCapabilities finds and returns a predefined StorageCapabilities for a given StorageClass
+func GetStorageCapabilities(cl client.Client, sc *storagev1.StorageClass) ([]StorageCapabilities, bool) {
 	provisionerKey := storageProvisionerKey(sc)
+	if provisionerKey == "kubernetes.io/no-provisioner" {
```

Comment:
As part of another task, I am trying to find a place of truth in the k8s libraries for this and `storageclass.kubernetes.io/is-default-class`. So far, no luck.
Doesn't block this PR, just thought you might have an idea

Reply:
It looks like it comes from [this package](https://pkg.go.dev/k8s.io/kubernetes/pkg/apis/storage/util)